### ISSUE TITLE
Some typos and ReLU activation for output layer

### ISSUE THREAD

**Issue by kabirahuja2431** [pixxelai/ai#2]
In lstm.py there are two small typos. In the import statement it should be from keras.callbacks import ModelCheckpoint instead of from keras.callbacks import ModelCheckPoint. And while compiling loss should be equal to losses.mean_squared_error instead of losses.squared_mean_error.

Finally in the output layer of cnn.py and lstm.py I think we should not use any activation function.

Thanks.
